src/tracker.jl

```diff
@@ -0,0 +1,33 @@
+using .Tracker: Tracker
+
+"""
+    TrackerBackend
+
+AD backend that uses reverse mode with Tracker.jl.
+"""
+struct TrackerBackend <: AbstractReverseMode end
+
+function second_lowest(::TrackerBackend)
+    return throw(ArgumentError("Tracker backend does not support nested differentiation."))
```

Comment:
This is the best way I could think of to signal that Tracker should not be used for higher order AD.